frigate/detectors/plugins/tensorrt.py

```diff
@@ -293,6 +293,16 @@ def detect_raw(self, tensor_input):
         # raw_detections: Nx7 numpy arrays of
         #             [[x, y, w, h, box_confidence, class_id, class_prob],
 
+        # throw out any detections with negative class IDs
+        valid_detections = []
+        for r in raw_detections:
+            if r[5] >= 0:
+                valid_detections.append(r)
+            else:
+                logger.debug(f"Found TensorRT detection with invalid class id {r}")
```

Comment:
Should this be a warning?

Reply:
for now probably, it's not clear if this is expected like how openvino doesn't treat it as an issue, but would be good to make it a warning so we can see how often it is happening